content/enterprise_influxdb/v1.9/concepts/schema_and_data_layout.md

```diff
@@ -9,136 +9,160 @@ menu:
+  - [Use recommended naming conventions](#use-recommended-naming-conventions)
+  - [Shard Group Duration Management](#shard-group-duration-management)
 
-## General recommendations
+## Where to store data (tag or field)
 
-### Encouraged schema design
+Your queries should guide what data you store in [tags](/enterprise_influxdb/v1.9/concepts/glossary/#tag) and what you store in [fields](/enterprise_influxdb/v1.9/concepts/glossary/#field) :
 
-We recommend that you:
+- Store commonly-queried and grouping ([`group()`](/flux/v0.x/stdlib/universe/group) or [`GROUP BY`](/enterprise_influxdb/v1.9/query_language/explore-data/#group-by-tags)) metadata in tags.
+- Store data in fields if each data point contains a different value.
+- Store numeric values as fields ([tag values](/enterprise_influxdb/v1.9/concepts/glossary/#tag-value) only support string values).
 
-- [Encode meta data in tags](#encode-meta-data-in-tags)
-- [Avoid using keywords as tag or field names](#avoid-using-keywords-as-tag-or-field-names)
+## Avoid too many series
 
-#### Encode meta data in tags
+IndexDB indexes the following data elements to speed up reads:
 
-[Tags](/enterprise_influxdb/v1.9/concepts/glossary/#tag) are indexed and [fields](/enterprise_influxdb/v1.9/concepts/glossary/#field) are not indexed.
-This means that queries on tags are more performant than those on fields.
+- [measurement](/enterprise_influxdb/v1.9/concepts/glossary/#measurement)
+- [tags](/enterprise_influxdb/v1.9/concepts/glossary/#tag)
 
-In general, your queries should guide what gets stored as a tag and what gets stored as a field:
+[Tag values](/enterprise_influxdb/v1.9/concepts/glossary/#tag-value) are indexed and [field values](/enterprise_influxdb/v1.9/concepts/glossary/#field-value) are not.
+This means that querying by tags is more performant than querying by fields.
+However, when too many indexes are created, both writes and reads may start to slow down.
 
-- Store commonly-queried meta data in tags
-- Store data in tags if you plan to use them with the InfluxQL `GROUP BY` clause
-- Store data in fields if you plan to use them with an [InfluxQL](/enterprise_influxdb/v1.9/query_language/functions/) function
-- Store numeric values as fields ([tag values](/enterprise_influxdb/v1.9/concepts/glossary/#tag-value) only support string values)
+Each unique set of indexed data elements forms a [series key](/enterprise_influxdb/v1.9/concepts/glossary/#series-key).
+[Tags](/enterprise_influxdb/v1.9/concepts/glossary/#tag) containing highly variable information like unique IDs, hashes, and random strings lead to a large number of [series](/enterprise_influxdb/v1.9/concepts/glossary/#series), also known as high [series cardinality](/enterprise_influxdb/v1.9/concepts/glossary/#series-cardinality).
+High series cardinality is a primary driver of high memory usage for many database workloads.
+Therefore, to reduce memory consumption, consider storing high-cardinality values in fields rather than in tags.
```

Comment:
Looks like my suggestion above needs copying to the other md files?

Reply:
@lesam It's in 1.8 schema-design.md. For all other versions, it's in resolve-high-cardinality.md

Reply:
I meant this one:

```suggestion
Therefore, to reduce memory consumption, consider storing high-cardinality values in fields values rather than in tags or field keys.
```

Reply:
And in the other md files if I missed any